src/protocols/bdx/BdxMessages.h

```diff
@@ -139,6 +147,9 @@ struct TransferInit : public BdxMessage
     BitFlags<TransferControlFlags> TransferCtlOptions;
     uint8_t Version = 0; ///< The highest version supported by the sender
 
+    // Range Control
+    BitFlags<RangeControlFlags> mRangeCtlFlags;
```

Comment:
The rest of the struct members are using PascalCase right now, but also I learned today that the convention is to use camelCase. So please do one or the other.